documentation/modules/collected-logs-cr-info.adoc

```diff
@@ -19,24 +19,19 @@ The `must-gather` tool collects the following logs and CR files in an archive fi
 ** `Plan` CR: Defines the VMs and storage and network mapping.
 
 * Logs:
-** `importer` pod: Disk-to-data-volume conversion log.
-+
-The pod naming convention is `importer-<migration_plan>-<vm_id>-<5_char_id>` for VMware and `importer-<migration_plan>-<partial_vm_id><5_char_id>` for {rhv-short}, for example, `importer-mig-plan-ed90dfc6-9a17-4a8btnfh`, where `ed90dfc6-9a17-4a8` is the partial VM ID and `btnfh` is the 5-character ID.
-
-** `conversion` pod: VM conversion log. The `conversion` pod runs `virt-v2v`, which installs and configures device drivers on the PVCs of the VM.
-+
-The pod naming convention is `<migration_plan>-<vm_id>-<5_char_id>` for VMware and `<migration_plan>-<partial_vm_id><5_char_id>` for {rhv-short}.
+** `importer` pod: Disk-to-data-volume conversion log. The `importer` pod naming convention is `importer-<migration_plan>-<vm_id><5_char_id>`, for example, `importer-mig-plan-ed90dfc6-9a17-4a8btnfh`, where `ed90dfc6-9a17-4a8` is a truncated {rhv-short} VM ID and `btnfh` is the generated 5-character ID.
+** `conversion` pod: VM conversion log. The `conversion` pod runs `virt-v2v`, which installs and configures device drivers on the PVCs of the VM. The `conversion` pod naming convention is `<migration_plan>-<vm_id><5_char_id>`.
 
```

Comment:
I think the hyphen is a part of the naming convention that gets removed in RHV pod name.
My suggestion is:

The pod naming convention is `importer-<migration_plan>-<vm_id>-<5_char_id>
If the combined migration plan name and VM ID are very long, the pod name might be truncated to make room for the generated 5-character ID. for example...


And the same for conversion pod

Reply:
I think the hyphen before the 5-char ID is only part of the VMware pod name, while it is not part of the RHV pod name.